watertap/unit_models/reverse_osmosis_base.py

```diff
@@ -737,6 +802,13 @@ def calculate_scaling_factors(self):
         if iscale.get_scaling_factor(self.recovery_vol_phase) is None:
             iscale.set_scaling_factor(self.recovery_vol_phase, 1)
 
+        if self.config.transport_model == TransportModel.SKK:
+            if iscale.get_scaling_factor(self.alpha) is None:
+                iscale.set_scaling_factor(self.alpha, 1e-8)
```

Comment:
I am still getting accustomed to alpha. Is it typically on the order of 1e8? Do we have a decent idea of how much it could vary?

Reply:
So alpha is approx 1/B and since B is initialized as 1e-8 I think initializing alpha as 1e8 is a good assumption. I don't think it's necessarily quite that simple since Menachem's students talked about characterizing membranes by alpha instead of B, but I think it is a good starting point.